torch/csrc/jit/python_ir.cpp

```diff
@@ -328,6 +330,9 @@ void initPythonIRBindings(PyObject * module_) {
   m.def("_jit_get_graph", [](tracer::TracingState* s) {
     return s->graph;
   });
+  m.def("import_ir_graph", [](const std::string& serialized_graph, std::vector<at::Tensor> & initializers) {
```

Comment:
If `initializers` are an output argument, then I think that changes to it don't carry over to Python.

Reply:
I thought I checked, but admittedly I was doing my tests in `libtorch` first.
In any case, I changed this function to return a `tuple`.

Reply:
Yep, it will mutate it in C++, but it won't propagate to Python.